pkg/import/import_differ.go

```diff
@@ -33,8 +35,19 @@ type ImportDiffer struct {
 	StackRefGetter StackRefGetter
 }
 
-func (id *ImportDiffer) DiffLifecycle(oldImg string, newImg string) (string, error) {
-	return id.Differ.Diff(oldImg, newImg)
+func (id *ImportDiffer) DiffLifecycle(kpConfig config.KpConfig, oldImg string, newImg string) (string, error) {
```

Comment:
It seems that the other methods in in here DiffClusterStore/DiffClusterStack/etc are actually reading the fully qualified digest from the registry so the lack of a similar workflow feels out of place here.

Reply:
These other calls do feel awkward because the other differs use StackRefGetter/StoreRefGetter which seem to be "too knowledgeable" about the intended usecase of an image (whether it is a stack/buildpack/etc).

Reply:
Perhaps this all could be simplified if we don't need to give the relocated images "pretty names" and we can write them all to the same repo in the registry.